# HG changeset patch
# User Rik <rik@octave.org>
# Date 1470694863 25200
#      Mon Aug 08 15:21:03 2016 -0700
# Node ID 42456fc9bf6c0217fe855d64fc25b98fac86c32b
# Parent  20e0c0b8820c9c4094f018b54426e2b258c07654
glpk.m: Style fixes for cset 20e0c0b8820c.

* glpk.m: Use space between function name and opening parenthesis.

diff --git a/scripts/optimization/glpk.m b/scripts/optimization/glpk.m
--- a/scripts/optimization/glpk.m
+++ b/scripts/optimization/glpk.m
@@ -499,17 +499,17 @@ function [xopt, fmin, errnum, extra] = g
     error ("glpk: A cannot be an empty matrix");
   endif
   if (! isreal (A))
     error ("glpk: A must be real valued, not %s", typeinfo (A));
   endif
   [nc, nxa] = size (A);
   if (nxa != nx)
     error ("glpk: A must be %d by %d, not %d by %d",
-           nc, nx, rows(A), columns(A));
+           nc, nx, rows (A), columns (A));
   endif
 
   ## 3) RHS
 
   if (isempty (b))
     error ("glpk: B cannot be an empty vector");
   endif
   if (! isreal (b) || length (b) != nc)
